iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/RemoveSubstation.java

```diff
@@ -0,0 +1,53 @@
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.modification.AbstractNetworkModification;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.Substation;
+import com.powsybl.iidm.network.VoltageLevel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static com.powsybl.iidm.modification.util.ModificationReports.notFoundSubstationReport;
+import static com.powsybl.iidm.modification.util.ModificationReports.removedSubstationReport;
+
+/**
+ * @author Maissa Souissi <maissa.souissi
+ * at rte-france.com>
+ */
+public class RemoveSubstation extends AbstractNetworkModification {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RemoveSubstation.class);
+
+    private final String substationId;
+
+    public RemoveSubstation(String substationId) {
```

Comment:
Remove public

Reply:
Why do you want to remove the public ? 
We have a public constructor for all Remove modifications.

Reply:
A new PR will be opened in order to modify the accessors and make them coherent in all the modifications (remove and create)